.changeset/config.json

```diff
@@ -7,5 +7,5 @@
   "access": "public",
   "baseBranch": "main",
   "updateInternalDependencies": "patch",
-  "ignore": ["@example/*"]
```

Comment:
@Mist3rBru we include this because we don't publish any packages within `@examples/` and all examples have namespaces.

Reply:
I noticed that all examples would copy-paste the same template, so I group then on examples folder and tested Changeset, it still ignores the examples, if you prefer to keep that I can revert it.

Reply:
I like this actually, as I was thinking of it more. I think we could leverage this to be used in within https://clack.cc to switch between different examples.

Reply:
That would be flat out nice!